specification/network/resource-manager/Microsoft.Network/stable/2021-01-01/customIpPrefix.json

```diff
@@ -363,6 +363,26 @@
           "type": "string",
           "description": "The prefix range in CIDR notation. Should include the start address and the prefix length."
         },
+        "SignedMessage": {
+          "type": "string",
+          "description": "Signed message for WAN validation."
+        },
+        "AuthorizationMessage": {
+          "type": "string",
+          "description": "Authorization message for WAN validation."
+        },
+        "CustomIpPrefixParent": {
+          "$ref": "#/definitions/CustomIpPrefix",
+          "description": "The Parent CustomIpPrefix for IPv6 /64 CustomIpPrefix."
+        },
```

Comment:
Please add examples for these property.

Reply:
Hi I fixed the error. But it seems there is a validation failure not related to my code.

Reply:
> Hi I fixed the error. But it seems there is a validation failure not related to my code.

Ignore it

Reply:
All the example are response message. We don't want AuthorizationMessage and SignedMessage to be shown in the response message. That's why we don't add them in the example.

Reply:
> All the example are response message. We don't want AuthorizationMessage and SignedMessage to be shown in the response message. That's why we don't add them in the example.

If AuthorizationMessage and SignedMessage are used in request body, you should add them in examples too. We are using tools to generate tests based on examples. If them are missed, our tests will not cover these properties.

Reply:
I just updated. For CustomIpPrefixParent, only ipv6 /64 has this property. So some example resource doesn't have it.